cli/cmd/env/set.go

```diff
@@ -38,6 +36,11 @@ func SetCmd(ch *cmdutil.Helper) *cobra.Command {
 				}
 			}
 
+			key, value, err := envValidator.ParseAndValidate(keyValPair)
+			if err != nil {
+				return err
+			}
```

Comment:
Could `Flags().StringToStringVar` ([docs](https://pkg.go.dev/github.com/spf13/pflag#StringToStringVar)) be used for this instead?